src/cryptography/hazmat/backends/openssl/ciphers.py

```diff
@@ -78,12 +78,19 @@ def __init__(self, backend, cipher, mode, operation):
                 len(iv_nonce), self._backend._ffi.NULL
             )
             self._backend.openssl_assert(res != 0)
-            if operation == self._DECRYPT:
-                res = self._backend._lib.EVP_CIPHER_CTX_ctrl(
-                    ctx, self._backend._lib.EVP_CTRL_GCM_SET_TAG,
-                    len(mode.tag), mode.tag
-                )
-                self._backend.openssl_assert(res != 0)
+            if operation == self._DECRYPT and \
+                    self._backend.openssl_version_number() < 0x10002000:
```

Comment:
I don't see a bug here, but LibreSSL reports its openssl_version_number as 0x20000000. It just so happens that LibreSSL doesn't have this bug, so this works, but it still makes me uncomfortable. I don't know that I have a better answer though. Thoughts @alex?